src/tests/EdFi.DataManagementService.Tests.E2E/Features/RESTAPIConventions/RESTAPIConventions.feature.disabled

```diff
@@ -0,0 +1,83 @@
+Feature: Authenticated REST API Conventions
```

Comment:
All of these scenarios are already covered in the existing Resource API CRUD features. This file is unnecessary.

Reply:
Removed